app/javascript/src/Policies/components/CustomPolicy.jsx

```diff
@@ -2,37 +2,22 @@
 
 import * as React from 'react'
 import { useState } from 'react'
-import { UnControlled as CodeMirror } from 'react-codemirror2'
 import SchemaForm from 'react-jsonschema-form'
-import { fromJsonString, toJsonString } from 'Policies/util'
-import type { Policy } from 'Policies/types/Policies'
+import { SchemaEditor } from 'Policies/components/SchemaEditor'
+import type { Policy, Schema } from 'Policies/types/Policies'
 import type {InputEvent} from 'Policies/types'
 import 'codemirror/mode/javascript/javascript'
 import 'Policies/styles/policies.scss'
 
 type OnChange = (InputEvent) => void
 
-const CM_OPTIONS = {
-  theme: 'default',
-  height: 'auto',
-  viewportMargin: Infinity,
-  mode: {
-    name: 'javascript',
-    json: true,
-    statementIndent: 2
-  },
-  lineNumbers: true,
-  lineWrapping: true,
-  indentWithTabs: false,
-  tabSize: 2
-}
-
 const POLICY_TEMPLATE: Policy = {
   schema: {
     '$schema': 'http://apicast.io/policy-v1/schema#manifest#',
-    'name': '[Name of the policy]',
-    'summary': '[A brief description of what it does.]',
-    'version': '[0.0.1]',
+    'name': 'Name of the policy',
+    'summary': 'A brief description of what it does.',
```

Comment:
```suggestion
    'summary': 'A one-line summary of what this policy does.',
```

Reply:
It should be less than 75 characters. Probably good to add this indication